presidio-anonymizer/presidio_anonymizer/anonymizers/__init__.py

```diff
@@ -0,0 +1,8 @@
+__all__ = [
```

Comment:
This should contain the actual imports, e.g.:
```
from .fpe import FPE
```
and the `__all__` should include `FPE` and not `fpe`

Other than that, it looks great!